### DISCUSSION TITLE
Need to understand how to edit main.yml

### DISCUSSION THREAD

**Discussion by Xitija** [project-sunbird/sunbird-devops#3872] [Q&A] [answered]
Hello All,

I am trying to enable invite-only feature for course batches.
For that I need to enable those APIs from the backend.

I got to know from this [discussion regarding Invite-only](https://github.com/orgs/Sunbird-Lern/discussions/141) that,
I need to modify the `kubernetes/ansible/roles/kong-api/defaults/main.yml` by adding a few entries to it.

I have cloned the DevOps repository and tried to edit that file, but I am not understanding how it works as it shows changes in that file on path or code, even when I have not made any changes.

![symlink](https://github.com/project-sunbird/sunbird-devops/assets/65657373/f2ee0660-55a7-4118-89ab-a81db37f1ad5)

I am not able to figure it out, I am totally new to this.
I request you to please guide me here.

@keshavprasadms @gandham-santhosh @coolbung @Renuka-S

**Comment by Nithin-ns24** [comment, marked as answer]
@Xitija the changes which you are trying to make the directory is soft linked to https://github.com/Nithin-ns24/sunbird-devops/blob/release-4.9.0_RC2/ansible/roles/kong-api/defaults/main.yml this directory and you can make changes in these path.
